providers/aws/config/config.go

```diff
@@ -13,7 +13,7 @@ var Config = plugin.Provider{
 	Name:            "aws",
 	ID:              "go.mondoo.com/cnquery/providers/aws",
 	Version:         "9.0.0",
-	ConnectionTypes: []string{provider.ConnectionType},
+	ConnectionTypes: []string{provider.DefaultConnectionType, provider.SshConnectionType, provider.RegistryImageConnectionType},
```

Comment:
Just came across this. This seems wrong. The handled types should be in the provider which can actually take care of these.

I discovered k8s container images and was then redirected to the aws provider, although the os provider should handle the container images.